interfaces/builtin/network_setup_control.go

```diff
@@ -45,6 +45,10 @@ const networkSetupControlConnectedPlugAppArmor = `
 # Netplan uses busctl internally, so allow using that as well
 /usr/bin/busctl ixr,
 
+# from systemd 254 onward, busctl binds to a unix socket upon startup to
+# facilitate debugging
+unix (bind) type=stream addr="@[0-9a-fA-F]*/bus/busctl/*"
```

Comment:
I think this is fine but I suspect we may want to have something like it in more places.

Reply:
aah, yes, good point, let me push one more patch for network-setup-observe

Reply:
also missed a `,` at the end here